examples/jsm/renderers/webgpu/WebGPURenderer.js

```diff
@@ -780,7 +780,7 @@ class WebGPURenderer {
 			const positionAttribute = geometry.attributes.position;
 			const vertexCount = ( drawRange.count !== Infinity ) ? drawRange.count : positionAttribute.count;
 
-			passEncoder.draw( vertexCount, 1, firstVertex, 0 );
+			passEncoder.draw( vertexCount, geometry.instanceCount || 1, firstVertex, 0 );
 
 			info.update( object, vertexCount );
```

Comment:
It's necessary to update `WebGPUInfo` to report correct number of rendered primitives.

Reply:
Thanks, updated. Please double check.